client/src/components/search/searchBar/search-bar-optimized.tsx

```diff
@@ -56,14 +62,19 @@ const SearchBarOptimized = ({
               value={value}
               ref={inputElementRef}
             />
-            <button className='ais-SearchBox-submit' type='submit'>
+            <button
+              className='ais-SearchBox-submit'
+              type='submit'
+              aria-label={t('search.submit')}
```

Comment:
I don't think we need to add aria-label here. The Magnifier already includes sr-only text for this button which is "Submit search terms". There is no need to override that with aria-label.